src/backend/gl/src/command.rs

```diff
@@ -153,11 +153,15 @@ impl From<c::Limits> for Limits {
 /// `display_fb` field.
 #[derive(Clone)]
 pub struct RawCommandBuffer {
-    pub(crate) memory: Arc<RefCell<pool::OwnedBuffer>>,
+    pub(crate) memory: Arc<pool::SharedBuffer>,
     pub(crate) buf: BufferSlice,
     // Buffer id for the owning command pool.
     // Only relevant if individual resets are allowed.
     pub(crate) id: u64,
+    // Associated pool and shared buffer is accessible.
+    // Mainly for safety reasons, not necessary but otherwise could result in weird crashes.
+    accessible: Arc<AtomicBool>,
```

Comment:
Is there a reason not to put the atomic inside `SharedBuffer` with `Mutex`/`RefCell`-like operations ?

Reply:
For `Individual` we have a HashMap for `SharedBuffer` and I want to protect the whole map and not the individual allocations, that's why I decided to move it out.